src/scripts/main.js

```diff
@@ -1,3 +1,257 @@
+  });
+});
+
+tbody.addEventListener('click', (e) => {
+  const selectedRow = e.target.closest('tr');
+
+  if (!selectedRow) {
+  }
+
+  tbody.querySelectorAll('tr').forEach((row) => {
+    row.classList.remove('active');
+  });
+
+  selectedRow.classList.add('active');
+});
+
+const form = document.createElement('form');
+
+form.className = 'new-employee-form';
+
+function createInput(labelText, inputName, dataQa, inputType = 'text') {
+  const label = document.createElement('label');
+
+  label.textContent = labelText + ': ';
+
+  const input = document.createElement('input');
+
+  input.name = inputName;
+  input.type = inputType;
+  input.setAttribute('data-qa', dataQa);
+  input.required = true;
+
+  label.appendChild(input);
+
+  return label;
+}
+
+form.appendChild(createInput('Name', 'name', 'name'));
+form.appendChild(createInput('Position', 'position', 'position'));
+
+const selectLabel = document.createElement('label');
+
+selectLabel.textContent = 'Office: ';
+
+const select = document.createElement('select');
+
+select.name = 'office';
+select.setAttribute('data-qa', 'office');
+select.required = true;
+
+const officeOptions = [
+  'Tokyo',
+  'Singapore',
+  'London',
+  'New York',
+  'Edinburgh',
+  'San Francisco',
+];
+
+officeOptions.forEach((option) => {
+  const opt = document.createElement('option');
+
+  opt.value = option;
+  opt.textContent = option;
+
+  select.appendChild(opt);
+});
+
+selectLabel.appendChild(select);
+
+form.appendChild(selectLabel);
+
+form.appendChild(createInput('Age', 'age', 'age', 'number'));
+form.appendChild(createInput('Salary', 'salary', 'salary', 'number'));
+
+const submitButton = document.createElement('button');
+
+submitButton.type = 'submit';
+
+submitButton.textContent = 'Save to table';
+
+form.appendChild(submitButton);
+
+document.body.append(form);
+
+function showNotification(message, isError = false) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.className = isError ? 'error' : 'success';
+  notification.classList.add('notification');
+
+  const title = document.createElement('h2');
+  const description = document.createElement('p');
+
+  title.classList.add('title');
+  title.textContent = isError ? 'Error' : 'Success';
+  description.textContent = message;
+  notification.append(title, description);
+  document.body.append(notification);
+
+  setTimeout(() => {
+    notification.style.display = 'none';
+  }, 5000);
+}
+
+function formatCurrency(value) {
+  return new Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 0,
+  }).format(value);
+}
+
+submitButton.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const nameEmployee = form.name.value;
+  const position = form.position.value;
+  const office = form.office.value;
+  const age = parseFloat(form.age.value, 10);
```

Comment:
The `parseFloat` function does not take a second argument. The second argument, `10`, is unnecessary here and should be removed.